.circleci/config.yml

```diff
@@ -19,14 +20,17 @@ jobs:
           command: |
             git submodule init
             git submodule update --remote
+      - run:
+          name: install helm
+          command: |
+            curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+            chmod 700 get_helm.sh
+            ./get_helm.sh
```

Comment:
we're using gcp-gcr orb already, so no?

Reply:
I was specifically suggesting running [`helm/install-helm-client`](https://circleci.com/orbs/registry/orb/circleci/helm#commands-install-helm-client) rather than this block since Circle will maintain that installer moving forward

Reply:
...and I'm not sure if that would actually succeed on `redhat-openshift/machine-for-local-cluster` executor